zerver/tests/test_auth_backends.py

```diff
@@ -1472,6 +1475,31 @@ def test_social_auth_invited_as_admin_but_expired(self) -> None:
         created_user = get_user_by_delivery_email(email, realm)
         self.assertEqual(created_user.role, UserProfile.ROLE_MEMBER)
 
+    def do_email_change(self, user_profile: UserProfile, action_key: str,
+                        different_account: bool=False) -> HttpResponse:
```

Comment:
Why do we need the `different_account` argument, rather than the caller passing a different user in the `user_profile` argument instead?

Reply:
Ohh, right, that argument doesn't make sense. Thanks, removed.